### ISSUE TITLE
Create the new major version docs

### ISSUE THREAD

**Issue by stronk7** [moodlehq/moodle-plugin-ci#286]
Each time that a new major release comes, there are a bunch of docs to create and/or amend.

As reference, these are changes that we did for v4: https://github.com/moodlehq/moodle-plugin-ci/commit/fcfc276236820d30a69fd45c25348abea55183b4

**Comment by ewallah** [comment]
Perhaps the sample scripts could be updated so the latest version is always used:
```
- composer create-project -n --no-dev --prefer-dist moodlehq/moodle-plugin-ci ci ^4
+ composer create-project -n --no-dev --prefer-dist moodlehq/moodle-plugin-ci ci
```

**Comment by gjb2048** [comment]
@ewallah But wouldn't that take away control during the migration period between two versions and not allow scheduling of changes to the configuration files?

**Comment by stronk7** [comment]
Correct, it's way better to have the version always specified, and then opt-in into the new one when ready.

Major versions come with incompatible changes:
- PHP requirements.
- Moodle requirements.
- Removed commands.
- ...

And that would break near everywhere. So pretty much the opposite, everybody should be, always, specifying the `moodle-plugin-ci` version to use. IMO.

Ciao :-)
